### PR TITLE
Update community page

### PR DESCRIPTION
**Pull request by dcalvin**
### What is changed?

- Updated Community page
- Move contributing page out of docs

### CODE REVIEW

**Comment by dcalvin** [comment]
See https://deploy-preview-177--tikv.netlify.app/community/community/ for preview.

File: content/community/community.md [hunk 1]
```diff
@@ -3,11 +3,47 @@ title: Community
 description: The TiKV community
 ---
 
-TiKV is an open source project with an active community. TiKV's source code is hosted on [GitHub](https://github.com/tikv/tikv) and is a [Cloud Native Computing Foundation](https://www.cncf.io) incubating project. We have information about how to [contribute here](/docs/3.0/reference/contribute/).
+TiKV is an open source project with an active community. TiKV's source code is hosted on [GitHub](https://github.com/tikv/tikv) and is a [Cloud Native Computing Foundation](https://www.cncf.io) incubating project. We have information about how to contribute [here](/community/contribute/).
 
-The TiKV community is organised into special interest groups (SIGs). More information about the community and its governance is in the [community repository](https://github.com/tikv/community). The current roster of SIGs is below, follow the links for more information.
+## SIGs
+
+The TiKV community is organized into special interest groups (SIGs). More information about the community and its governance is in the [community repository](https://github.com/tikv/community). The current roster of SIGs is listed below. follow the links for more information.
```
Comment (nrc) [inline content/community/community.md:10]: "The current roster of SIGs is listed below." This is not quite right, prefer either "The current roster of SIGs is below." or "The current SIGs are listed below."

File: content/community/community.md [hunk 2]
```diff
@@ -3,11 +3,47 @@ title: Community
...
 
 * [Coprocessor](/community/sig-coprocessor/)
 * [Engine](/community/sig-engine/)
 * [Raft](/community/sig-raft/)
 * [Transaction](/community/sig-transaction/)
+
+## Communications
+
+In TiKV community we value open communication and open collaboration. Here are the general communication channels we use to communicate.
```
Comment (nrc) [inline content/community/community.md:19]:
```suggestion
In the TiKV community we value open communication and collaboration. Here are our general communication channels.
```

File: content/community/community.md [hunk 3]
```diff
@@ -3,11 +3,47 @@ title: Community
...
+
+- **General contact methods**
+
+    - [Slack](https://join.slack.com/t/tikv-wg/shared_invite/enQtNTUyODE4ODU2MzI0LWVlMWMzMDkyNWE5ZjY1ODAzMWUwZGVhNGNhYTc3MzJhYWE0Y2FjYjliYzY1OWJlYTc4OWVjZWM1NDkwN2QxNDE) (Recommended)
+    - [Twitter](https://twitter.com/tikvproject)
+    - [Reddit](https://www.reddit.com/r/TiKV)
```
Comment (nrc) [inline content/community/community.md:25]: We should not promote Reddit - the forum is basically unused and Reddit has a poor reputation

File: content/community/community.md [hunk 4]
```diff
@@ -3,11 +3,47 @@ title: Community
...
+
+- **Report security vulnerabilities**
+
+    To report a security vulnerability, please send an email to [TiKV-security](tikv-security@lists.cncf.io) group.
+
+- **Community monthly meeting**
+
+  The TiKV team meets on the 4th Wednesday of every month (unless otherwise specified) at 07.00 p.m. PST. See [Meeting notes](https://docs.google.com/document/d/1CWUAkBrcm9KPclAu8fWHZzByZ0yhsQdRggnEdqtRMQ8/edit) for details.
+
+## Projects built upon TiKV
```
Comment (nrc) [inline content/community/community.md:36]:
```suggestion
## Projects built on TiKV
```

File: content/community/community.md [hunk 5]
```diff
@@ -3,11 +3,47 @@ title: Community
...
+
+As an open-source and vendor-neutral project, TiKV thrives by collaborating with the wider community. Here are are some projects that are built upon TiKV:
```
Comment (nrc) [inline content/community/community.md:38]:
```suggestion
TiKV is an open-source and vendor-neutral project, it thrives by collaborating with the wider community. Here are are some projects that are built using TiKV:
```

File: content/community/community.md [hunk 6]
```diff
@@ -3,11 +3,47 @@ title: Community
...
+
+- [Tidis](https://github.com/yongman/tidis) -  a distributed NoSQL database, providing a Redis protocol API (string, list, hash, set, sorted set)
+- [Titan](https://github.com/meitu/titan) - a distributed implementation of Redis compatible layer based on TiKV
+- [Titea](https://github.com/gengmei-tech/titea) - a distributed redis protocol compatible NoSQL Database, providing a Redis protocol, base on Tikv and PD
```
Comment (nrc) [inline content/community/community.md:42]:
```suggestion
- [Titea](https://github.com/gengmei-tech/titea) - a distributed redis protocol compatible NoSQL Database, providing a Redis protocol, based on Tikv and PD
```

File: content/community/community.md [hunk 7]
```diff
@@ -3,11 +3,47 @@ title: Community
...
+- [TiPrometheus](https://github.com/bragfoo/TiPrometheus) - A adapter/gateway for TiKV
```
Comment (nrc) [inline content/community/community.md:43]:
```suggestion
- [TiPrometheus](https://github.com/bragfoo/TiPrometheus) - An adapter/gateway for TiKV
```

Should this description mention Prmetheus or data collection or something?

File: content/community/community.md [hunk 8]
```diff
@@ -3,11 +3,47 @@ title: Community
...
+- [TiBigData](https://github.com/pingcap-incubator/TiBigData) - Misc BigData components for TiDB
+- [Zetta](https://github.com/zhihu/zetta) - an open-source NoSQL database that supports Transaction and Cloud Spanner like API
```
Comment (nrc) [inline content/community/community.md:45]:
```suggestion
- [Zetta](https://github.com/zhihu/zetta) - an open-source NoSQL database that supports transactions and a Cloud Spanner-like API
```

**Pull request merged**